docs/source/install.rst

```diff
@@ -65,4 +66,4 @@ Follow the steps above to start the Koku API with Docker Compose
 
 Start masu services: ``masu_branch> make docker-up`` 
 To trigger Masu to process data, send a GET request to http://<baseUrl>:5000/api/cost-management/v1/download/
-You may want see Masu logs using `docker logs koku_worker -f`
\ No newline at end of file
+You may want see Masu logs using `docker logs koku_worker -f`
```

Comment:
Grammar suggestions:
You may want to see Masu logs using `docker logs koku_worker -f`